completion_aggregator/api/v0/views.py

```diff
@@ -326,7 +326,10 @@ def get(self, request, course_key):
             raise NotFound()
         requested_fields = self.get_requested_fields()
         enrollment = UserEnrollments(self.user).get_course_enrollment(course_key)
-        aggregator_queryset = self.get_queryset().filter(course_key=course_key)
+        aggregator_queryset = self.get_queryset().filter(
```

Comment:
I'm not sure why we need to fetch the enrollment here and use the ``course_id`` and ``user`` from that, would those not be the same as ``course_key`` and ``self.user``?

Reply:
Only if aggregators exist.  If the user hasn't interacted with the course, we still need to report that they have made 0% progress.  I believe @pomegranited came up with this strategy.

Reply:
Oh.  I just noticed that this is on the detail view.  I was thinking of different code on the list view, to get results for all users in the course.

And that we've already checked that the enrollment exists right above.  So I'm not sure why it's here.